### PR TITLE
run-a-node: Add section on Running with Non-root User to Prerequisites

### PR DESCRIPTION
**Pull request by tjanez**
_NOTE: For easier reviewing, review commits separately._

### CODE REVIEW

**Review by pro-wh — APPROVED** [review]
ya

File: docs/general/run-a-node/prerequisites/system-configuration.md [hunk 1]
```diff
@@ -57,3 +60,132 @@ If you are running Oasis Node via [Docker](https://www.docker.com/) you can pass
...
+
+2. Stop your Oasis Node.
+
+3. Transfer ownership of the datadir to the `oasis` user:
+
+```sh
+chown -R oasis /serverdir/node
+```
+
+See https://github.com/oasisprotocol/docs/blob/main/docs/general/run-a-node/troubleshooting.md#invalid-permissions for more information.
```
Comment (pro-wh) [inline docs/general/run-a-node/prerequisites/system-configuration.md]: https://docs.oasis.dev/general/run-a-node/troubleshooting#invalid-permissions

File: docs/general/run-a-node/prerequisites/system-configuration.mdx [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+USER oasis
+...
+```
+</TabItem>
+
+<TabItem value="runit">
+
+Wrap the invocation in a [`chpst` command]:
+```sh
+chpst -u oasis oasis-node
```
Comment (pro-wh) [inline docs/general/run-a-node/prerequisites/system-configuration.mdx]: I had `oasis-node ...` to indicate the usual arguments follow

File: docs/general/run-a-node/prerequisites/system-configuration.md (hunk 1 above)
Comment (pro-wh) [reply docs/general/run-a-node/prerequisites/system-configuration.md]: or is there some special syntax for internal links

File: docs/general/run-a-node/prerequisites/system-configuration.mdx (hunk 2 above)
Comment (tjanez) [reply docs/general/run-a-node/prerequisites/system-configuration.mdx]: Thanks for the explanation. Fixed.

File: docs/general/run-a-node/prerequisites/system-configuration.md (hunk 1 above)
Comment (tjanez) [reply docs/general/run-a-node/prerequisites/system-configuration.md]: Yeah, one just links to the docs via internal path.

**Comment by pro-wh** [comment]
> 2:37:59 PM: - On source page path = /general/run-a-node/upgrade-log:
2:37:59 PM:    -> linking to prerequisites/system-configuration.mdx#change-to-non-root (resolved as: /general/run-a-node/prerequisites/system-configuration.mdx)

**Pull request merged**